yarn.lock

```diff
@@ -649,24 +649,28 @@
     eslint-plugin-sonarjs "^0.5.0"
     prettier "^2.1.2"
 
-"@pagopa/io-functions-commons@^20.3.6":
-  version "20.3.6"
-  resolved "https://registry.yarnpkg.com/@pagopa/io-functions-commons/-/io-functions-commons-20.3.6.tgz#e648c7f3d161210860d8a5eb18e43ca747817654"
-  integrity sha512-zGXldfkkcR4oH6vdNLAkDbQHwN4dCdTvvuscg6RBhxWHEx1ZAM0lKhkVrqT5/FxYKwkAQj+bYuUuJEC7NH3KXA==
+"@pagopa/express-azure-functions@^2.0.0":
+  version "2.0.0"
+  resolved "https://registry.yarnpkg.com/@pagopa/express-azure-functions/-/express-azure-functions-2.0.0.tgz#eb52a0b997d931c1509372e2a9bea22a8ca85c17"
+  integrity sha512-IFZqtk0e2sfkMZIxYqPORzxcKRkbIrVJesR6eMLNwzh1rA4bl2uh9ZHk1m55LNq4ZmaxREDu+1JcGlIaZQgKNQ==
+
+"@pagopa/io-functions-commons@^20.5.3":
```

Comment:
Could we check if this upgrade contains breaking changes that could affect io-functions-admin clients?

Reply:
There have been no breaking changes, just new features and dependencies bumps.